cmd/repo-init/repo_manager.go

```diff
@@ -200,10 +199,19 @@ func commitChanges(message, email, name string) error {
 	if err := bumper.Call(os.Stdout, os.Stderr, "git", "add", "-A"); err != nil {
 		return fmt.Errorf("git add: %w", err)
 	}
-	commitArgs := []string{"commit", "-m", message}
-	if name != "" && email != "" {
-		commitArgs = append(commitArgs, "--author", fmt.Sprintf("%s <%s>", name, email))
+
+	if err := bumper.Call(os.Stdout, os.Stderr, "git", "config", "--local", "user.email", email); err != nil {
```

Comment:
Is this what this PR is about?
Can we set this up as a git globe setting, more properly in the image that this tool runs?
We probably did this already in our commonly used os image.

We have many tools run the git-push after git-commit.
I do not see anyone need git-config.
Do you notice the failure in production or locally?

Reply:
Yes, this is the actual change in this PR. I thought about setting it in the image, but this way allows for the authorized user themselves to appear as the committer. I actually found the pattern in https://github.com/openshift/ci-tools/blob/master/pkg/github/prcreation/prcreation.go#L139-L148